core/src/test/java/io/parsingdata/metal/expression/value/reference/LocalReferenceNestedScopeTest.java

```diff
@@ -77,9 +79,9 @@ public void testNestedScopesCalculated() {
         nestedScopes(eq(nth(ref("left"), sub(count(ref("left")), count(ref("right"))))));
     }
 
-//    @Test
-//    public void testNestedScopesLocalReferenced() {
-//        nestedScopes(eq(last(lref("left"))));
-//    }
+    @Test
+    public void testNestedScopesLocalReferenced() {
+        nestedScopes(eq(first(scope(ref("left"), con(0)))));
```

Comment:
![lebowski](https://user-images.githubusercontent.com/501609/55646170-cae4f380-57da-11e9-9a04-837a6d5e3fe5.gif)

Reply:
I'm a little hesitant to make this resolved since I like scrolling past it so much :sweat_smile: